Rakefile

```diff
@@ -20,7 +20,11 @@ task :setup => ["bundler:checkout"] do
 end
 
 Rake::TestTask.new do |t|
-  t.ruby_opts = %w[--disable-gems]
+  # if bundler is in RubyGems repo, require it
```

Comment:
Do not inject bundler library on `RUBY_OPTS` layer. It can be changed with `ENV['DONT_USE_BUNDLER_FOR_GEMDEPS']`

Reply:
Locally, I made the change to `ENV['DONT_USE_BUNDLER_FOR_GEMDEPS']` and I have a repeatable error:
```

  1) Error:
TestGemCommandsSetupCommand#test_env_shebang_flag:
Errno::ENOENT: No such file or directory @ rb_sysopen - C:/Greg/temp/test_rubygems_6828/install/bin/bundle
    C:/Greg/GitHub/rubygems/test/rubygems/test_gem_commands_setup_command.rb:123:in `read'
    C:/Greg/GitHub/rubygems/test/rubygems/test_gem_commands_setup_command.rb:123:in `test_env_shebang_flag'
```

Reply:
It's not related to this.

I already fixed ruby core repo: https://github.com/ruby/ruby/commit/b9212a7771a5e67f3b48bbf4c76cd087a8d87e49

Reply:
Good morning.  Sorry, do you mean the problem in general or the TestGemCommandsSetupCommand#test_env_shebang_flag error?